Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,7 +6,7 @@
         "repositoryURL": "https://github.com/ecosia/ios-core.git",
         "state": {
           "branch": "main",
-          "revision": "6098b2de91d43917743ed1883eced133d07108f2",
+          "revision": "7ef7ba234816b909f801574389a72c32f88fd7d4",
```

Comment:
I suggest that for the next release we create another tag for core and hardwire this one.. but not needed now 👍

Reply:
The core release is already in place but as this was pointed to main I opted not to change it.